sites/frontend/components/Header/Nav/SubNav/SubNavList/SubNavListItem/SubNavButton.js

```diff
@@ -59,7 +61,13 @@ const SubNavButton = styled('button')(
 );
 SubNavButton.displayName = 'SubNavButton';
 
-export default ({ pillar, showSecondaryNav, toggleSecondaryNav }) => {
+type Props = {
+    pillar: PillarType,
+    showSecondaryNav: boolean,
+    toggleSecondaryNav: Function,
+};
+
+export default ({ pillar, showSecondaryNav, toggleSecondaryNav }: Props) => {
```

Comment:
Do we want to add return types (`React.Node`) to stateless components? I guess it makes sense

Reply:
it ought to be able to infer that shoudn' tit?

Reply:
Yeah I guess. So do we have rules around what types to make explicit and what's safe to assume the compiler can infer?

Reply:
If a function doesn't have any branching, it doesn't need to specify a return type?